package.json

```diff
@@ -2,15 +2,17 @@
   "private": true,
   "dependencies": {},
   "devDependencies": {
+    "@babel/core": "^7.0.0",
     "@babel/plugin-transform-modules-commonjs": "^7.9.0",
     "@testing-library/jest-dom": "^4.0.0",
     "@testing-library/react": "^9.1.0",
     "@testing-library/react-hooks": "^2.0.0",
+    "axios": "0.9.0",
```

Comment:
Can you upgrade the version to `"axios": "^0.19.2",` so it matches the `strapi-admin` one?

Reply:
Sure thing!